test/features/organisations.feature

```diff
@@ -1,53 +1,35 @@
 @users
 Feature: Organization APIs
 
-    Scenario: Organisation membership is accessible to sysadmins
-        Given "SysAdmin" as the persona
+    Scenario Outline: Organisation membership is accessible to admins of the organisation
+        Given "<User>" as the persona
         When I log in
         And I view the "department-of-health" organisation API "including" users
         Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "organisation_admin"') and contains(string(), '"name": "editor"')]"
 
-    Scenario: Organisation membership is accessible to admins of the organisation
-        Given "Organisation Admin" as the persona
-        When I log in
-        And I view the "department-of-health" organisation API "including" users
-        Then I should see an element with xpath "//*[contains(string(), '"success": true,') and contains(string(), '"name": "organisation_admin"') and contains(string(), '"name": "editor"')]"
+        Examples: Admins
+            | User                |
```

Comment:
User an admin?

Reply:
This is the variable name for the step to use.